library/src/methods/merge/merge.ts

```diff
@@ -16,36 +16,27 @@ type ObjectSchemas = [
   ...ObjectSchema<any>[]
 ];
 
-/**
```

Comment:
In these cases, I prefer to keep the commented overflow signatures and leave the implementation uncommented. Can we configure that?

Reply:
Could you find a solution for this?

Reply:
> I prefer to keep the commented overflow signatures and leave the implementation uncommented.
Yes!
 By configuring like [this](https://github.com/kazizi55/valibot/blob/3d36da387c8febeab5f4d79b1ba3b3d8e2241b4b/library/.eslintrc.cjs#L52-L60), this can be implemented 😄 
For overloaded functions, we only have to write one JSDoc at the top.

Reply:
Thank you! I will have a look at the changes in the evening or tomorrow.